hack/csv-generate.sh

```diff
@@ -1,26 +1,17 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
?

Reply:
Running a command through `/usr/bin/env` has the benefit of looking for whatever the default version of the program is in your current environment, see https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash for more details. We have some trouble with using `#!/bin/bash` under the KubeVirt project, so I just change it to a more flexible version.

Reply:
thanks, this should be wide available knowledge 
We should also have that in all scripts, right?

Reply:
yep, I will create a separate PR for this.